packages/components/src/ClickableStyle/ClickableStyle.module.css

```diff
@@ -172,6 +177,35 @@
   }
 }
 
+.variantText {
+  color: var(--primary-color);
+  border: transparent;
+
+  &:focus,
+  &.stateFocus {
+    background-color: var(--tertiary-color);
+    color: var(--primary-color);
+  }
+
+  &:hover,
+  &.stateHover {
+    background-color: var(--tertiary-color);
+    color: var(--primary-color);
+  }
+
+  &:active,
+  &.stateActive {
+    /* TODO: primary is being set to 700 on click; should be 500 here */
+    background-color: var(--primary-color);
```

Comment:
isn't it actually `-800` now 😬   If these colors are only used in the text button, I'm tempted to call
- `tertiary-color` -> `text-hocus-bg-color`
- add a `text-active-bg-color` for the `-500` variants

it is an annoying large mix of colors, but at least it's clearer then not every variant needs that many

Reply:
Good idea! I replaced `tertiary` with `plain-background-color` and just tracked the background color there since, you're right, it's not being reused by any of the other variants.

> isn't it actually -800 now 😬 

yes and the click color is 600 so I was wrong all around lol